.github/pull_request_template.md

```diff
@@ -0,0 +1,66 @@
+<!--
+Thank you for contributing to TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+-->
+
+## What have you changed? (mandatory)
+
+<!--
+Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
+- Summarize your change (mandatory)
+- How does this PR work? Need a brief introduction for the changed logic (optional)
+- Separately describe each logical change and avoid lazy messages (optional)
+- Describe any limitations of the current code (optional)
+
+Please **NOTE** that:
+- Do not assume reviewers understand the original issue
+-->
+
+## What is the type of the changes? (mandatory)
+
+<!--
+The currently defined types are listed below, please pick one of the types for this PR by removing the others.
+-->
+
+- New feature (non-breaking change which adds functionality)
+- Improvement (non-breaking change which is an improvement to an existing feature)
+- Bug fix (non-breaking change which fixes an issue)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## How has this PR been tested? (mandatory)
+
+<!--
+Please describe the tests that you ran to verify your changes.
+Have you finished unit tests, integration tests, or manual tests?
+What additional tests would give you greater confidence in this change?
+-->
+
+## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
+
+<!--
+If there is document change, please file a PR in ([docs](https://github.com/pingcap/docs) or [docs-cn](https://github.com/pingcap/docs-cn)) and add the PR number here.
+-->
+
+## Does this PR affect tidb-ansible update? (mandatory)
+
+<!--
+If there is a configuration or metrics change, please file a PR in [tidb-ansible](https://github.com/pingcap/tidb-ansible), and add the PR number here.
+-->
+
+## Does this PR need to be added to the release notes? (mandatory)
+
+<!--
+If this PR needs to be added to the release notes, please:
+1. add the "**release-note**" label for this PR
+2. write your release note in the below block
+3. include the string "action required" if the PR requires additional action from users switching to the new release
+-->
+
+```release-note
+```
```

Comment:
The code cannot be displayed normally. Please move "release-note" to the next single line.